test/jdk/com/sun/jdi/OnThrowTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2015, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023 SAP SE. All rights reserved.
```

Comment:
Changing the copyright here seems wrong, nothing changed on this file.

Reply:
Oops.